.stylelintrc

```diff
@@ -0,0 +1,73 @@
+{
+  "plugins": [
+    "stylelint-order",
+    "stylelint-scss",
+    "stylelint-selector-bem-pattern"
+  ],
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-property-sort-order-smacss"
+  ],
+  "rules": {
+    "comment-empty-line-before": [
+      "always",
+      {
+        "except": ["first-nested"],
+        "ignore": ["after-comment", "stylelint-commands"]
+      }
+    ],
+    "declaration-empty-line-before": [
+      "always",
+      {
+        "except": ["first-nested"],
+        "ignore": ["after-declaration"]
+      }
+    ],
+
+    "order/order": [
+      [
+        "custom-properties",
+        "dollar-variables",
+        {
+          "type": "at-rule",
+          "name": "extend"
+        },
+        {
+          "type": "at-rule",
+          "name": "include",
+          "hasBlock": false
+        },
+        "declarations",
+        {
+          "type": "at-rule",
+          "hasBlock": true
+        },
+        {
+          "type": "at-rule",
+          "name": "include",
+          "hasBlock": true
+        },
+        {
+          "type": "rule",
+          "selector": "/^&:\\w/"
+        },
+        {
+          "type": "rule",
+          "selector": "/^&::\\w/"
+        },
+        "rules"
+      ]
+    ],
+
+    "plugin/selector-bem-pattern": {
+      "preset": "bem"
+    },
+
+    "scss/at-extend-no-missing-placeholder": true,
```

Comment:
I can't tell if these are defaults or overrides; can we just extend `stylelint-scss` and get rid of most/all of these?

Reply:
`stylelint-scss` is a plugin, not a configuration to extend—it just provides the extra Sass-specific rules.

Reply:
Extended a different config that includes the Sass-specific rules for us.